hw13/trigrams.py

```diff
@@ -0,0 +1,36 @@
+import random
+
+
+d = {}
```

Comment:
Try and avoid single character variable names. As programs become larger, it can become difficult to debug when you have no idea what "d" means.